content/agent/basic_agent_usage/heroku.md

```diff
@@ -63,6 +63,38 @@ Depending on your use case, you may want to set your hostname so that hosts are
 - The Datadog Agent configuration files are at `/app/.apt/etc/datadog-agent`
 - The Datadog Agent logs are at `/app/.apt/var/log/datadog`
 
+## Heroku Log Collection
+
+** This integration is currently available for the EU region only**
```

Comment:
we generally want to avoid use of "currently" and lowkey promising that something will eventually happen. what do you think @l0k0ms

Reply:
In this case, something will happen as it will be released in the US region as well :)